fastmri/data/mri_data.py

```diff
@@ -301,6 +314,27 @@ def __init__(
                 if ex[2]["encoding_size"][1] in num_cols  # type: ignore
             ]
 
+    def Download_csv(self, version):
+        url = 'https://cdn.jsdelivr.net/gh/microsoft/fastmri-plus@{0}/Annotations/knee.csv'.format(version)
+
+        if not os.path.isdir('.annotation'):
+            os.system('mkdir .annotation')
+
+        annotation_fileName = os.path.join(os.getcwd(), '.annotation', 'annotation_knee_{0}.csv'.format(version))
+        if not os.path.isfile(annotation_fileName):
+            # download csv from github and save it locally
+            try:
+                response = requests.get(url)
+            except requests.exceptions.RequestException as e:
+                raise SystemExit(e)
+
+            with open(annotation_fileName, 'w') as f:
+                writer = csv.writer(f)
+                for line in response.iter_lines():
+                    writer.writerow(line.decode('utf-8').split(','))
+            f.close()
```

Comment:
Could you deserialize using `pandas` instead of using `csv`? It reduces our dependencies and avoids an extra file handle.